src/jenkins_job_insight/main.py

```
@@ -70,6 +72,37 @@
+        The validated integer port number.
+
+    Raises:
+        SystemExit: If PORT is not a valid integer.
+    """
+    raw_port = os.environ.get("PORT", "8000")
+    try:
+        return int(raw_port)
+    except ValueError:
+        raise SystemExit(
+            f"Invalid PORT environment variable: {raw_port!r}. Must be an integer."
+        )
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)